raiden/tests/integration/test_balance_proof_check.py

```diff
@@ -0,0 +1,145 @@
+import pytest
+
+from raiden import waiting
+from raiden.api.python import RaidenAPI
+from raiden.constants import EMPTY_HASH, EMPTY_SIGNATURE
+from raiden.network.proxies import TokenNetwork
+from raiden.tests.utils.events import must_contain_entry
+from raiden.tests.utils.network import CHAIN
+from raiden.tests.utils.transfer import direct_transfer, get_channelstate
+from raiden.transfer import views
+from raiden.transfer.state_change import ContractReceiveChannelSettled
+
+
+@pytest.mark.parametrize('deposit', [10])
+@pytest.mark.parametrize('channels_per_node', [CHAIN])
+@pytest.mark.parametrize('number_of_nodes', [2])
+def test_invalid_close(
+        raiden_network,
+        number_of_nodes,
+        deposit,
+        token_addresses,
+        network_wait,
+):
+    app0, app1 = raiden_network
+    token_address = token_addresses[0]
+    chain_state = views.state_from_app(app0)
+    payment_network_id = app0.raiden.default_registry.address
+    registry_address = app0.raiden.default_registry.address
+    token_network_identifier = views.get_token_network_identifier_by_token_address(
+        chain_state,
+        payment_network_id,
+        token_address,
+    )
+    channel_identifier = get_channelstate(app0, app1, token_network_identifier).identifier
+
+    # make a transfer
+    direct_transfer(
+        app0,
+        app1,
+        token_network_identifier,
+        1,
+        timeout=network_wait * number_of_nodes,
+    )
+    # stop app1 - the test uses token_network_contract now
+    app1.stop()
+    token_network_contract = TokenNetwork(app1.raiden.chain.client, token_network_identifier)
+
+    # app1 closes the channel
```

Comment:
Add more info on the comment. 

```diff
- # app1 closes the channel
+ # app1 closes the channel with an empty hash instead of the expected hash of the transferred amount from app0
```